mobile/src/main/java/org/openhab/habdroid/ui/OpenHABWidgetAdapter.java

```diff
@@ -579,8 +579,14 @@ public void onStopTrackingTouch(SeekBar seekBar) {
                 break;
             case TYPE_SELECTION:
                 int spinnerSelectedIndex = -1;
-                if (labelTextView != null)
-                    labelTextView.setText(openHABWidget.getLabel());
+                splitString = openHABWidget.getLabel().split("\\[|\\]");
```

Comment:
Shouldn't it be `String[] splitString = ...`

Reply:
No because the variable is declared globally and used in several cases of the switch.

Reply:
ok, thats in line 115